portfolio/src/main/webapp/commonlogic.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
consider changing the file name to something more descriptive (e.g. sidebar, sidenav)

Reply:
Renamed to sidebar.html